EIPS/eip-2537.md

```diff
@@ -287,7 +287,7 @@ Discounts table as a vector of pairs `[k, discount]`:
 
 #### Pairing check operation
 
-The cost of the pairing check operation is `43000*k + 65000` where `k` is a number of pairs.
+The cost of the pairing check operation is `32587*k + 37698` where `k` is a number of pairs.
```

Comment:
Maybe let's round these to 32600 and 37700.

Reply:
Done, thx @chfast :)